zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/VersionSpecificTemplates.java

```diff
@@ -225,10 +226,10 @@ static float getVersion(HttpCall.Factory callFactory) throws IOException {
   enum ReadVersionNumber implements HttpCall.BodyConverter<Float> {
     INSTANCE;
 
-    @Override public Float convert(HttpData content) {
+    @Override public Float convert(JsonParser parser, Supplier<String> contentString) {
       // The version number is read only once per startup. Hence, there is less impact to allocating
       // strings. We retain the string so that it can be logged if the ES response is malformed.
-      String body = content.toStringUtf8();
+      String body = contentString.get();
```

Comment:
No big deal but think readTree accepts JsonParser too.

Reply:
it broke the api chain but I'll try to figure out why

Reply:
annoyingly :)

![Screenshot 2019-08-02 at 12 28 05 PM](https://user-images.githubusercontent.com/64215/62344542-01304680-b521-11e9-999d-18b7fbd05640.png)

Reply:
I avoided that api..